src/main/java/it/pagopa/pn/paperchannel/middleware/queue/consumer/handler/HandlersFactory.java

```diff
@@ -38,6 +38,7 @@ public class HandlersFactory {
 
     private ConcurrentHashMap<String, MessageHandler> map;
 
+
```

Comment:
gli handler aggiunti non sono associati in questo file, quindi non vengono chiamati

Reply:
si è trattato di un refuso. provvediamo a fixare subito